bedrock/urls.py

```diff
@@ -12,39 +15,36 @@
 
 # The default django 500 handler doesn't run the ContextProcessors, which breaks
 # the base template page. So we replace it with one that does!
-handler500 = 'bedrock.base.views.server_error_view'
 
+handler500 = 'bedrock.base.views.server_error_view'
 
-urlpatterns = (
-    # Main pages
-    url(r'^foundation/', include('bedrock.foundation.urls')),
-    url(r'^grants/', include('bedrock.grants.urls')),
-    url(r'^about/legal/', include('bedrock.legal.urls')),
-    url(r'^press/', include('bedrock.press.urls')),
-    url(r'^privacy/', include('bedrock.privacy.urls')),
-    url(r'^styleguide/', include('bedrock.styleguide.urls')),
-    url(r'^security/', include('bedrock.security.urls')),
-    url(r'', include('bedrock.firefox.urls')),
-    url(r'', include('bedrock.mozorg.urls')),
-    url(r'', include('bedrock.newsletter.urls')),
-    url(r'^etc/', include('bedrock.etc.urls')),
-    url(r'^exp/', include('bedrock.exp.urls')),
-
-    url(r'^healthz/$', watchman_views.ping, name="watchman.ping"),
-    url(r'^readiness/$', watchman_views.status, name="watchman.status"),
-    url(r'^healthz-cron/$', base_views.cron_health_check),
-    url(r'^csp-violation-capture$', base_views.csp_violation_capture,
-        name='csp-violation-capture'),
-    url(r'^country-code\.json$', base_views.geolocate,
-        name='geolocate')
-)
+urlpatterns = (  # Main pages
+    path('foundation/', include('bedrock.foundation.urls')),
+    path('grants/', include('bedrock.grants.urls')),
+    path('about/legal/', include('bedrock.legal.urls')),
+    path('press/', include('bedrock.press.urls')),
+    path('privacy/', include('bedrock.privacy.urls')),
+    path('styleguide/', include('bedrock.styleguide.urls')),
+    path('security/', include('bedrock.security.urls')),
+    path('', include('bedrock.firefox.urls')),
+    path('', include('bedrock.mozorg.urls')),
+    path('', include('bedrock.newsletter.urls')),
+    path('etc/', include('bedrock.etc.urls')),
+    path('exp/', include('bedrock.exp.urls')),
+    path('healthz/', watchman_views.ping, name='watchman.ping'),
+    path('readiness/', watchman_views.status, name='watchman.status'),
+    path('healthz-cron/', base_views.cron_health_check),
+    path('csp-violation-capture', base_views.csp_violation_capture,
+         name='csp-violation-capture'),
+    path('country-code.json', base_views.geolocate, name='geolocate'),
+    )
 
 if settings.DEBUG:
 
+
     def show404(request):
         return import_string(handler404)(request, '')
 
-    urlpatterns += (
-        url(r'^404/$', show404),
-        url(r'^500/$', import_string(handler500)),
-    )
+
+    urlpatterns += (path('404/', show404), path('500/',
+                    import_string(handler500)))
```

Comment:
I'd much rather keep the formatting for these lists like the following:

```suggestion
    urlpatterns += (
        path('404/', show404), 
        path('500/', import_string(handler500)),
    )
```